### DISCUSSION TITLE
Oven: Control for steam would be great

### DISCUSSION THREAD

**Discussion by ursolino69** [ekutner/home-connect-hass#437] [Ideas]
Quite a few ovens have a steam function which is great for baking bread!
Maybe I am just missing it. I do see the sensor Cooking.Oven.EnumType.AddedSteam but not the control.
![image](https://github.com/user-attachments/assets/3704508e-48f7-4e85-a342-91bc93cd8d04)
Thank you very much for this excellent integration!!!

**Comment by GivaniDurand** [comment]
+1 for me plz
Oven supports full steam including regenerate program
Neff B59FS3AY0

**Comment by Greentreegarden** [comment]
+1 from me - my oven has "Air Fry" which is not in the HA list.
"Air Fry" is included in the official Bosch Home Connect Android app though.
